openfisca_us/variables/gov/states/ny/tax/income/credits/refundable/ny_eitc.py

```diff
@@ -0,0 +1,17 @@
+from openfisca_us.model_api import *
+
+
+class ny_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "NY EITC"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.nysenate.gov/legislation/laws/TAX/606"  # (d)
+
+    def formula(tax_unit, period, parameters):
+        eitc = tax_unit("earned_income_tax_credit", period)
```

Comment:
OK if this isn't included in the net income tree, but when it is, we'll need to ensure the state code filters it (if the is_eligible logic isn't in by then).